_posts/blog/2018-10-11-4rios-review.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "QUEERING THE DIGITAL ARCHIVE: '4 RIOS' AND GRAPHIC NARRATION"
+modified:
+categories: blog
+excerpt: ...the unique construction of this archive calls into question institutionalized norms regarding what an archive is and how it functions.
+tags: []
+author: "Kaitlin Burns"
+image:
+	feature:
+---
+# **Queering the Method: "4 Ríos" and Archival Narration**
+
+4 Ríos Digital Archive. http://4rios.co. Acccessed 28 Sept. 2018.
```

Comment:
Please make the url a clickable link using markdown formatting. Instructions: http://www.halperta.com/criticalarchives/tutorial/writing-in-markdown/